war/src/main/js/filter-build-history.js

```diff
@@ -27,12 +27,19 @@ function updateBuilds() {
                 var dataTable = getDataTable(buildHistoryContainer);
                 var rows = dataTable.rows;
 
+                // Check there are no existing rows (except the search bar) before showing the no builds banner
+                if (rows.length <= 1 && rsp.responseText === "<table class=\"pane\"></table>") {
```

Comment:
seems a bit fragile =/

Reply:
Agreed, the widget itself is something I want to take another look at in the future so I'll likely update this in the future.